src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -1138,3 +1169,41 @@ func setClusterProperties(fs afero.Fs, cfgFileLocation string, properties map[st
 
 	return nil
 }
+
+func parseSandboxFlag(args []string, sandbox *string) {
+	for i := 0; i < len(args)-1; i++ {
+		if args[i] == "--sandbox" && !strings.HasPrefix(args[i+1], "-") {
+			*sandbox = args[i+1]
+		}
+	}
+}
+
+const helpSandbox = `Sandbox mode sets relevant properties for local development or test container
```

Comment:
use `"Sandbox mode uses well-known configuration properties for development or tests environments:"`